Apromore-Core-Components/Apromore-Manager/pom.xml

```diff
@@ -77,6 +62,8 @@
                             org.apromore.apmlog.impl;version="[1.0,2)",
                             org.apromore.cache.ehcache;version="[1.0,2)",
                             org.apromore.dao.*;version="[1.0,2)",
+                            org.apromore.portal.helper,
+                            org.apromore.portal.model,
                             *
                         </Import-Package>
                         <Export-Package>
```

Comment:
Can this PR review and clean up the Export-Package as well? The improper exported interfaces to consider:
org.apromore.helper: indirect as it is not provided by Apromore-Manager),
org.apromore.plugin.deployment.exception: indirect as it is not provided by Apromore-Manager),
org.apromore.plugin.provider.*: indirect as it is not provided by Apromore-Manager),
org.apromore.util.datetime: missing package